esx_service/vmdk_ops.py

```diff
@@ -837,25 +838,31 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     except ValidationError as ex:
         return err(str(ex))
 
-    if datastore and not vmdk_utils.validate_datastore(datastore):
+    if not datastore:
```

Comment:
Fixed and removed the new condition added. Its not needed for now and handles the rare case that a datastore is temporarily not visible on the host.